### DISCUSSION TITLE
context.contextActivities.parent

### DISCUSSION THREAD

**Discussion by sfraysse** [gaia-x-dases/xapi-virtual-classroom#3] [General]
I would suggest to remove the `context.contextActivities.parent` prop from the templates/examples when there is no relevant parent to set.

For example in templates such as `initialized` or `terminated`, the parent will depend of the context in which the virtual classroom is delivered. It may be a course, or nothing. As we don't know the context, we should remove it from the examples/templates. Adopters will be free to define a relevant parent for their use case. We may add a comment in the spec to recommend adding a relevant parent if there is one.

For some templates such as `posted`, the object is a message and the parent must be the virtual classroom activity. In this case, yes, it is mandatory and it should be in the examples/templates. So the `posted` template looks correct on this point for me.

**Comment by quitterie-lcs** [comment]
I would say that when it is a statement that describes an activity directly related to the virtual classroom, it should be specified. Your point makes sense. However, for objects that are embedded in the virtual classroom, it should be specified, (for messages and polls currently) as this profile defines all activities for the virtual classrooms and such activities like messages or polls could be embedded in other learning activities.

**Reply by sfraysse** [reply]
I think we agree. My point was to remove the parent from examples such as `initialized` or `terminated`, where the parent is not the virtual classroom.
